### ISSUE TITLE
Use of 'toLocaleString' when converting date values

### ISSUE THREAD

**Issue by meelash** [mrodrig/json-2-csv#146]
Wasn't sure if this would be seen in the closed issue, so, creating a new issue here:

I have another use case for this: https://github.com/mrodrig/json-2-csv/issues/64 , wondering if this is still a possibility to get implemented, and/or if there is a better way currently to accomplish what I'm seeking.
By default datetimes will be converted to UTC when creating a csv, since that is what Date.prototype.toString() does. Generally, I'm guessing this would not be desired behavior.

Example:
```
{
  startTime: new Date('2020-03-05 16:34:39')
}
```
If my local timezone is EST, json2csv output will be:

> startTime
> '2020-03-05 21:34:39'

Is there a way I can (even a temporary fix) get json2csv to use toLocaleString or any other way to get dates to convert to a string in the local timezone?

**Comment by mrodrig** [comment]
Hi @meelash, thanks for bringing this up again. The module has definitely been updated since then and I think it would be a bit easier to provide that functionality now, but I'd likely have to require an option in order to ensure backwards compatibility. The only other option that came to mind if you needed a quick solution for a script or something like that, would be to set the TZ environment variable to force Node to use the specified timezone. Something like this should cause the output dates to be in EST rather than UTC:
```
process.env.TZ = 'America/New_York';
// Convert json to csv
converter.json2csv(json, options)
  .then((csv) => { ... })
  .catch((error) => { ... });
``` 

Depending on where you're generating the CSV though (eg. Node web app), then it might also impact other areas of your app, like log timestamps, so it's something to consider.

I'll plan to start looking into the best way to allow for the use of `.toLocaleString()` sometime tomorrow.

**Comment by mrodrig** [comment]
Added the `useLocaleFormat` option, which when set to `true` will cause the module to use `toLocaleString()` instead of `toString()`. This new option is published in `3.7.0` and should be available on NPM.
